components/brave_today/browser/direct_feed_controller.cc

```diff
@@ -339,19 +387,20 @@ void DirectFeedController::OnResponse(
     std::move(callback).Run(std::move(result));
     return;
   }
-  // Reponse is valid, but still might not be a feed
-  FeedData data;
-  if (!parse_feed_string(::rust::String(body_content), data)) {
-    VLOG(1) << feed_url.spec() << " not a valid feed.";
-    VLOG(2) << "Response body was:";
-    VLOG(2) << body_content;
-    std::move(callback).Run(std::move(result));
-    return;
-  }
-  // Valid feed
-  result->success = true;
-  result->data = data;
-  std::move(callback).Run(std::move(result));
+
+  // Response is valid, but still might not be a feed
+  ParseFeedDataOffMainThread(std::move(feed_url), std::move(body_content),
```

Comment:
Is `std::move()` appropriate for `feed_url`? `feed_url` is a const reference of `DirectFeedController::OnResponse()`, which suggests to me that it should just be passed without `std::move()` so that a copy is made.

Reply:
That makes sense, yeah. Sorry, my C++ is pretty rusty